packages/api-bindings/codegen-api.yml

```diff
@@ -75,6 +78,24 @@ generates:
           skipTypename: true
           immutableTypes: false
           inlineFragmentTypes: combine
-      - 'typescript-react-apollo'
+          omitOperationSuffix: true
+          operationResultSuffix: 'Data'
+
+  src/graphql/hooks.ts:
```

Comment:
can we also add a comment on the very top of these two new files that they are auto-generated and to not make direct changes to these files (`src/graphql/operations.ts` and `src/graphql/hooks.ts`) as it is not obvious, compared to the previous `generated.tsx`.

Reply:
good call, done in the following PR: https://github.com/lens-protocol/lens-sdk/pull/255/commits/3b8f49847e69734923156ced24f2672a0a305b46